### PR TITLE
Test modified ssa properties

### PR DESCRIPTION
**Pull request by jdemon519**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 9b0dfc29abf6a41ef5005836d414762faf58212e:

`cfme/tests/containers/test_containers_properties.py`:
- :heavy_exclamation_mark: Line 21:1: **W293** *blank line contains whitespace*
- :bangbang: Line 25:1: **E302** *expected 2 blank lines, found 0*
- :bangbang: Line 54:1: **E302** *expected 2 blank lines, found 0*
- :bangbang: Line 80:1: **E302** *expected 2 blank lines, found 0*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d2f5631bae5985ce2ea170ac2aa25204c4fd6c5e:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/containers/image.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
 
+checkbox = ".//input[@class='list-grid-checkbox']"
```
Comment (mfalesni) [inline cfme/containers/image.py]: You sure this checkbox does not have a name or id?

File: cfme/tests/containers/test_containers_properties.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]: Unnecessary newline here

File: cfme/tests/containers/test_containers_properties.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+from cfme.containers.pod import Pod, list_tbl as list_tbl_pod
+from cfme.containers.route import Route, list_tbl as list_tbl_route
+from cfme.containers.project import Project, list_tbl as list_tbl_proj
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]: Unnecessary newline here.

File: cfme/tests/containers/test_containers_properties.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+""" Polarion test case CMP-9911.
+"""
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]: I believe this belongs as a test docstring (right after `def ...`) rather than being outside.

Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]: The same applies for the other comments.

File: cfme/tests/containers/test_containers_properties.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+        # verify that mgmt pods exist in ui listed pods
+        assert set(ui_pods).issubset(
+            [obj.name for obj in mgmt_objs]), 'Missing objects'
+
+    for name in ui_pods:
+        obj = Pod(name, provider)
+        field_content = getattr(obj.summary.properties, rel).text_value
+
+        if field_content:
+            assert len(field_content) != 0
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]: This assert is a tautology. the if will let you in only when the field_content is not empty (empty string is false) and then you are checking if the string is empty again, therefore this assert can never be false. :exclamation:

File: cfme/tests/containers/test_containers_properties.py [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        # verify that mgmt routes exist in ui listed routes
+        assert set(ui_routes).issubset(
+            [obj.name for obj in mgmt_objs]), 'Missing objects'
+
+    for name in ui_routes:
+        obj = Route(name, provider)
+        field_content = getattr(obj.summary.properties, rel).text_value
+
+        if field_content:
+            assert len(field_content) != 0
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]: Same here.

File: cfme/tests/containers/test_provider_crud.py [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(blockers=[1274842])
+def test_provider_edit_port(request, a_container_provider):
+    old_port = a_container_provider.port
+    request.addfinalizer(lambda: a_container_provider.update({'port': old_port}))
+    with update(a_container_provider):
+        a_container_provider.port = '1234'
+    assert str(a_container_provider.port) == a_container_provider.get_detail('Properties', 'Port')
```
Comment (mfalesni) [inline cfme/tests/containers/test_provider_crud.py]: You should be able to use summary here:
```python
assert int(a_container_provider.port) == a_container_provider.summary.properties.port.value
```

**Pull request closed without merging**